zstacklib/zstacklib/utils/iscsi_nl.py

```
@@ -0,0 +1,484 @@
+                        match = pattern.match(line)
+                        if match:
+                            iscsi_value = int(match.group(1))
+                            logger.debug("Found NETLINK_ISCSI = %d in %s", iscsi_value, path)
+                            return iscsi_value
+            except IOError as e:
+                logger.warning("Cannot read file %s: %s", path, e)
+
+    # If not found in common paths, traverse include directories
+    include_dirs = ["/usr/include", "/usr/local/include"]
+    for dir_path in include_dirs:
+        for root, dirs, files in os.walk(dir_path):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)